anime-movieset-list.xml

```diff
@@ -2505,4 +2505,13 @@
       <title type="main" xml:lang="x-jat">Oshiri Tantei Collection</title>
     </titles>
   </set>
+    <set>
+    <anime anidbid="14397">Seishun Buta Yarou wa Yumemiru Shoujo no Yume o Minai</anime>
+    <anime anidbid="17640">Seishun Buta Yarou wa Odekake Sister no Yume o Minai</anime>
+    <anime anidbid="17862">Seishun Buta Yarou wa Ransel Girl no Yume o Minai</anime>
+    <titles>
+      <title type="main" xml:lang="x-jat">Seishun Buta Yarou wa ... no Yume o Minai Collection</title>
+      <title type="official" xml:lang="en">Rascal Does Not Dream of ... Collection</title>
```

Comment:
Not big on the naming either, at least the romaji one. The English one fits.
Maybe use "Ao Buta", the short name for the series instead?

Reply:
Hmm, `Ao Buta Movie Collection` ?

Reply:
That ("Ao Buta"), "Seibuta", cut the title after `wa` without the ending, or simply keep the original series' full title instead (`bunny girl senpai`) as the movies are also referred as "BGS Movie 1-3" and that might be easier to recognize.
![image](https://github.com/Anime-Lists/anime-lists/assets/7104931/0873e132-45a2-48a8-9623-2328159170f9)

Reply:
Ok, something like this
```
<title type="main" xml:lang="x-jat">Seishun Buta Yarou wa Bunny Girl Senpai no Yume o Minai Collection</title>
<title type="official" xml:lang="en">Rascal Does Not Dream of Bunny Girl Senpai Collection</title>
```
I guess, it could be worse, heh

Reply:
Sure, that works too. The ellipsis kinda worked on English, since the different part from the titles is at the end it doubled as a separator from the "Collection" qualifier.
On the romaji it just looked weird splitting the title in the middle.